tests/conftest.py

```diff
@@ -1,6 +1,9 @@
 # pylint: disable=missing-function-docstring,missing-module-docstring
 from typeguard import typechecked
-from typeguard.importhook import install_import_hook
+try:
+    from typeguard.importhook import install_import_hook
+except ImportError:
+    from typeguard import install_import_hook
```

Comment:
This should make black happy again
```suggestion

try:
    from typeguard.importhook import install_import_hook
except ImportError:
    from typeguard import install_import_hook
```